drizzlepac/haputils/catalog_utils.py

```diff
@@ -299,15 +307,17 @@ def compute_background(self, box_size, win_size,
             self.bkg_rms_ra = np.full_like(imgdata, bkg_rms)
             self.bkg_median = bkg_median
             self.bkg_rms_median = bkg_rms
+            self.negative_threshold = -1.0 * bkg_rms
 
         # BACKGROUND COMPUTATION 3 (Background2D)
         # The simple_bkg = True is the way to force the background to be computed with the
         # sigma-clipped algorithm, regardless of any other criterion. If simple_bkg == True,
         # the compute_background() is done, otherwise try to use Background2D to compute the background.
         if not simple_bkg and not is_zero_background_defined:
 
-            # If the sigma-clipped background image skew is less than the threshold,
-            # compute a two-dimensional background fit.
+            # If the sigma-clipped background image skew is greater than the threshold,
+            # compute a two-dimensional background fit.  A larger skew implies
+            # more sources in the field, which requires a more complex background.
             if bkg_skew < bkg_skew_threshold:
```

Comment:
I believe the comment, and currently the bkg_skew_threshold = 0.5.  The code matches my notes from our discussion long ago, but the code seems backwards for when to compute the more complex 2D background fit. It seems the code should be 

if bkg_skew > bkg_skew_threshold:

Reply:
Reversed in 9e1e7ed.  The logic for applying the skew in determining what background model to use was also inverted, so larger skews end up using 2D background and lower skews use sigma-clipped backgrounds.